### ISSUE TITLE
problem in timesformer_pytorch.py

### ISSUE THREAD

**Issue by Weizhongjin** [lucidrains/TimeSformer-pytorch#15]
start from line 182
`video = rearrange(video, 'b f c (h p1) (w p2) -> b (f h w) (p1 p2 c)', p1 = p, p2 = p)`
i think this should be 
`video = rearrange(video, 'b f c (hp p1) (wp p2) -> b (f hp wp) (p1 p2 c)', p1 = p, p2 = p)`

**Comment by lucidrains** [comment]
I believe that is equivalent

**Comment by tcapelle** [comment]
yeah, variables are silent inside this type of expressions. You could call `h` or `z` and it doe snot matter.
